src/pages/Profiles.vue

```diff
@@ -650,15 +603,22 @@ export default class Profiles extends Vue {
     }
 
     async updateProfileList() {
-        this.profileList = ["Default"];
         const profilesDirectory: string = this.activeProfile.getDirectory();
-        await fs.readdir(profilesDirectory).then(dirContents => {
-            dirContents.forEach(async (file: string) => {
-                if ((await fs.stat(path.join(profilesDirectory, file))).isDirectory() && file.toLowerCase() !== 'default' && file.toLowerCase() !== "_profile_update") {
-                    this.profileList = [...this.profileList, file].sort();
-                }
+        let profilesDirectoryContents = ['Default'];
```

Comment:
This seems unnecessary since the value is straight out overwritten in the try-block and not used outside of the try-block at all.